README.md

```diff
@@ -11,7 +11,7 @@ Follow the videos in this and next lessons to implement the page block by block
 - Start writing styles in the `main.scss` file. It is be explained in the `Sass` lesson
 - **DON'T** try to do it `Pixel Perfect` - implement it the most `simple` way so it looks similar;
 - When done check yourself using the [CHECKLIST](./checklist.md) when finished;
-- Deploy and create a Pull Request with a [DEMO LINK](https://<your_account>.github.io/layout_landing-page/)
+- Deploy and create a Pull Request with a [DEMO LINK](https://<NazarShpylovyi>.github.io/layout_landing-page/)
```

Comment:
Fix this link and add it to the PR description 
<img width="999" alt="image" src="https://github.com/user-attachments/assets/28698797-1e74-4078-9f40-9d364d1e836a" />
```suggestion
- Deploy and create a Pull Request with a [DEMO LINK](https://NazarShpylovyi.github.io/layout_landing-page/)
```